backend/db/ddl-12-cset_members_items_plus.jinja.sql

```diff
@@ -5,7 +5,7 @@ SELECT csmi.*
         , c.concept_name
         , c.concept_class_id
 FROM {{schema}}cset_members_items csmi
-JOIN concept c ON csmi.concept_id = c.concept_id);
+JOIN {{schema}}concept c ON csmi.concept_id = c.concept_id);
```

Comment:
## Bug fix: `cset_members_item_plus` sometimes dropping
What happened: When the backend tests run, it does `remake_test_schema()`. However, since `{{schema}}` was missing here, when it created this view, it used the schema of the connection, which was set to `test_schema`. So, even though `cset_members_item_plus` was being inserted into `n3c`, it was _dependent on `test_n3c.concept`_. Thus, the next time `remake_test_schema()` ran, during the step at the beginning where it drops w/ cascade `test_n3c`, it resulted in `n3c.cset_members_items_plus` being dropped as well.